src/index.html

```diff
@@ -2,11 +2,447 @@
+      href="./styles/main.scss"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__content">
+          <div class="logo">
+            <a
+              href="#home"
+              class="logo__link"
+            >
+              <img
+                src="/images/logo.png"
+                alt="Moyo logo"
+                class="logo__image"
+              >
+            </a>
+          </div>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#apple"
+                  class="nav__link is-active"
+                >
+                  Apple
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#samsung"
+                  class="nav__link"
+                >
+                  Samsung
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#smartphones"
+                  class="nav__link"
+                >
+                  Smartphones
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#laptops & computers"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#gadgets"
+                  class="nav__link"
+                >
+                  Gadgets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#tablets"
+                  class="nav__link"
+                >
+                  Tablets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#photo"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#video"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__description">
+          Product code: 195434
+        </p>
+
+        <p class="card__rating">
```

Comment:
I understand why you do that, but it's not a good idea to do that. Look, `p` - is a tag which used for text paragraphs. You used this tag not for text, you use it for an appearance that gives you p-tag. It's not text data, it's rating block with images. So, will be better if you use here `div` tags. You will write more styling, but on the other hand, structure will be more logical

Reply:
Hi! 👋 It was previously so. But in morning classes [Misha recommended ](https://mateacademy.slack.com/files/U054NT2QQKE/F05A291EZ63/image.png)changing div in that case for semantic tag p, so I refactored) Do I need to change it back as it was?